quic-encryption-offload.md

```diff
@@ -73,10 +83,11 @@ typedef enum _QEO_CIPHER_TYPE {
 } QEO_CIPHER_TYPE;
 
 typedef struct _QEO_CONNECTION {
-    BOOLEAN IsAdd : 1;
-    BOOLEAN IsTransmit : 1;
-    BOOLEAN RESERVED : 6;
-    QEO_CIPHER_TYPE CipherType;
+    uint32_t Operation  : 1;  // QEO_OPERATION
```

Comment:
Isn't the description for `Operation` and `Direction` missing?
It is sort of self-describing, but still.
Also, remove descriptions for `IsAdd` and `IsTransmit`.

Reply:
Good catch. Fixed.